core/src/test/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImplTest.java

```diff
@@ -114,7 +114,7 @@ public void givenPlayerIsInitialised_whenVideoIsPrepared_thenCancelsTimeout() {
         }
 
         @Test
-        public void givenPlayerIsInitialised_whenVideoHasError_thenCancelsTimeout() {
+        public void givenPlayerIsInitialised_whenVideoHasError_thenPlayerResourcesAreReleased_andNotListeners() {
```

Comment:
Given the change we have done it seems like we should check that `listenersHolder.getStateChangedListeners().onVideoStopped()` is not called on error? 

Same for the other test